src/seismometer/__init__.py

```diff
@@ -72,3 +72,46 @@ def run_startup(
     # Surface api into namespace
     s_module = importlib.import_module("seismometer._api")
     globals().update(vars(s_module))
+
+
+def download_example_dataset(dataset_name: str, branch_name: str = "main"):
+    """
+    Downloads an example dataset from the specified branch to local data/ directory.
+
+    Args:
```

Comment:
Can you change the docstring format to numpy-style?

Instead of Args it's Parameters like:
```python
    Parameters
    ----------
```
Bigger thing is to add this function to one of the .rst under docs/reference/ and make sure it shows up consistently

Reply:
fixed